x-pack/plugins/ml/public/explorer/explorer.html

```diff
@@ -115,22 +154,34 @@ <h4 class="euiTitle euiTitle--small">No {{swimlaneViewByFieldName}} influencers
           <br /><br />
         </div>
   
-        <span class="panel-title euiText">
-          Anomalies
-        </span>
+        <span
+          class="panel-title euiText"
+          18n-id="xpack.ml.explorer.anomaliesTitle"
+          i18n-default-message="Anomalies"
+        ></span>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive ml-anomalies-controls">
           <div class="euiFlexItem euiFlexItem--flexGrowZero" style="width:170px">
             <div class="euiFormRow" id="select_severity_control">
-              <label class="euiFormLabel" for="select_severity">Severity threshold</label>
+              <label
+                class="euiFormLabel"
+                for="select_severity"
+                18n-id="xpack.ml.explorer.severityThresholdLabel"
+                i18n-default-message="Severity threshold"
+              ></label>
               <div class="euiFormControlLayout">
                 <ml-select-severity id="select_severity" />
               </div>
             </div>
           </div>
           <div class="euiFlexItem euiFlexItem--flexGrowZero" style="width:170px">
             <div class="euiFormRow" id="select_interval_control">
-              <label class="euiFormLabel" for="select_interval">Interval</label>
+              <label
+                class="euiFormLabel"
+                for="select_interval"
+                18n-id="xpack.ml.explorer.intervalLabel"
+                i18n-default-message="Interval"
+              ></label>
```

Comment:
This doesn't get rendered correctly, the string "Interval" doesn't end up as inner HTML/text of the label element.

![image](https://user-images.githubusercontent.com/230104/50635151-47083a80-0f51-11e9-954b-9247a94a596e.png)

![image](https://user-images.githubusercontent.com/230104/50635164-57201a00-0f51-11e9-9aa9-b19b40563fff.png)